<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogStrDialog</class>
 <widget class="QWidget" name="LogStrDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log Streams</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2" colspan="3">
    <widget class="QLabel" name="Label5">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="cBStream1C">
     <property name="toolTip">
      <string>Enable rover data stream logging</string>
     </property>
     <property name="text">
      <string>(6) Rover</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="Label6">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Option</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QComboBox" name="cBStream2">
     <property name="toolTip">
      <string>Select log stream type for base station</string>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="2" colspan="5">
    <widget class="QComboBox" name="cBStream3">
     <property name="toolTip">
      <string>Select log stream type for correction data</string>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QPushButton" name="btnStream3">
     <property name="toolTip">
      <string>Select options for the correction log data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QPushButton" name="btnStream2">
     <property name="toolTip">
      <string>Select options for the base station log data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QPushButton" name="btnStream1">
     <property name="toolTip">
      <string>Select options for the rover log data stream</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QCheckBox" name="cBTimeTag">
     <property name="toolTip">
      <string>Generate time-tags</string>
     </property>
     <property name="text">
      <string>Time-Tag</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="cBStream2C">
     <property name="toolTip">
      <string>Enable base station data stream logging</string>
     </property>
     <property name="text">
      <string>(7) Base Station</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="cBStream3C">
     <property name="toolTip">
      <string>Enable correction data stream logging</string>
     </property>
     <property name="text">
      <string>(8) Correction</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblF1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Log File Paths</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="Label10">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Log Stream</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QComboBox" name="cBStream1">
     <property name="toolTip">
      <string>Select log stream type for rover</string>
     </property>
     <item>
      <property name="text">
       <string>Serial</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Client</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TCP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Server</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NTRIP Caster</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>File</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="8">
    <widget class="QCheckBox" name="cBOutputEvent">
     <property name="text">
      <string>Output Event</string>
     </property>
    </widget>
   </item>
   <item row="5" column="8">
    <widget class="QPushButton" name="btnFile1">
     <property name="toolTip">
      <string>Select rover log data file</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="6" column="8">
    <widget class="QPushButton" name="btnFile2">
     <property name="toolTip">
      <string>Select base station log data file</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="7" column="8">
    <widget class="QPushButton" name="btnFile3">
     <property name="toolTip">
      <string>Select correction log data file</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="8">
    <widget class="QLineEdit" name="lEFilePath1">
     <property name="toolTip">
      <string>Specify file name for rover log data</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="8">
    <widget class="QLineEdit" name="lEFilePath2">
     <property name="toolTip">
      <string>Specify file name for base station log data</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="8">
    <widget class="QLineEdit" name="lEFilePath3">
     <property name="toolTip">
      <string>Specify file name for correction log data</string>
     </property>
    </widget>
   </item>
   <item row="8" column="8">
    <widget class="QPushButton" name="btnCancel">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="7">
    <widget class="QPushButton" name="btnOk">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>&amp;OK</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QPushButton" name="btnKey">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QLabel" name="lblH">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>H</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QComboBox" name="cBSwapInterval">
     <property name="toolTip">
      <string>Select swap interval for output data. Use specific keys (see &quot;?&quot; button) in the file path to generate unique filenames.</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.25</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>12</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>24</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2" colspan="2">
    <widget class="QLabel" name="lblSwapInterval">
     <property name="toolTip">
      <string/>
     </property>
     <property name="text">
      <string>Swap Interval</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cBStream1C</tabstop>
  <tabstop>cBStream1</tabstop>
  <tabstop>btnStream1</tabstop>
  <tabstop>cBOutputEvent</tabstop>
  <tabstop>cBStream2C</tabstop>
  <tabstop>cBStream2</tabstop>
  <tabstop>btnStream2</tabstop>
  <tabstop>cBStream3C</tabstop>
  <tabstop>cBStream3</tabstop>
  <tabstop>btnStream3</tabstop>
  <tabstop>lEFilePath1</tabstop>
  <tabstop>btnFile1</tabstop>
  <tabstop>lEFilePath2</tabstop>
  <tabstop>btnFile2</tabstop>
  <tabstop>lEFilePath3</tabstop>
  <tabstop>btnFile3</tabstop>
  <tabstop>cBTimeTag</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
